docs/graphql/manual/guides/production-checklist.rst

```diff
@@ -0,0 +1,76 @@
+.. _production-checklist:
+
+Production checklist
+====================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+This guide is a checklist for configuring and securing GraphQL engine for a production deployment.
+
+Security
+--------
+
+The following steps are recommended to better secure your production GraphQL engine instance:
+
+Admin secret
+^^^^^^^^^^^^
+
+Configure an admin secret as described :doc:`here <../deployment/securing-graphql-endpoint>`.
+
+Reverse proxy & API gateway
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You might want to set up Hasura behind a load balancer to improve performance, reliability and security.
+
+NOTE: SHAHIDH WILL ADD MORE INFO HERE INCLUDING CONFIGURATION
+
+nginx
+~~~~~
+
+`Here <https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/>`_ is how you can configure an NGINX reverse proxy.
+
+Caddy
+~~~~~
+
+`Here <https://caddyserver.com/docs/proxy>`_ is how you can configure a Caddy proxy.
+
+
+CORS domains
+^^^^^^^^^^^^
+
+Configure your CORS domains as described `here <https://docs.hasura.io/1.0/graphql/manual/deployment/graphql-engine-flags/config-examples.html#configure-cors>`_.
+
+API security
```

Comment:
We should mention that people should review their authorization rules by going to the authorization role summary page. Especially they should double check permissions given to any anonymous type roles.